CMakeLists.txt

```diff
@@ -17,13 +17,20 @@
 
 cmake_minimum_required(VERSION 3.5)
 
+# build the sdk targets
+project(olp-cpp-sdk VERSION 0.6.0)
+
 # Options
 option(EDGE_SDK_ENABLE_TESTING "Flag to enable/disable building unit and integration tests" ON)
 option(EDGE_SDK_BUILD_DOC "Build SDK documentation" OFF)
 option(EDGE_SDK_NO_EXCEPTION "Turn off EXCEPTION" OFF)
 option(EDGE_SDK_BUILD_EXTERNAL_DEPS "Download and build external dependencies" ON)
 option(EDGE_SDK_BUILD_EXAMPLES "Enable examples targets" OFF)
 
+if (MSVC)
```

Comment:
Do we need to guard config variables by platform? They should have no effect either way.

Reply:
Removed the guard